docs/en/core/tables.md

```diff
@@ -45,11 +45,11 @@ Obviously, writing Markdown tables is sometimes unavoidable, but always painful.
 
 While Zettlr maintains a strict policy on [WYSIWYG](https://en.wikipedia.org/wiki/WYSIWYG) ("What You See Is What You Get"), as it tends to blur your perception of what you are actually writing, tables are exempt from this policy. Because even though the [WYSIWYM](https://en.wikipedia.org/wiki/WYSIWYM) ("What You See Is What You Mean") approach is superior to WYSIWYG, there's simply no use in applying this to tables.
 
-Therefore, to aid you with writing tables, Zettlr is now shipping with a powerful table editor that brings to you everything you need to not completely lose your mind while creating tables in your documents. With the table editor, the above-mentioned table looks like this:
+Therefore, to aid you with writing tables, Zettlr is now shipping with a powerful table editor that brings to you everything you need to not completely lose your mind while creating tables in your documents. With the table editor, the above mentioned table looks like this:
 
 ![A table as displayed by the table editor](../img/zettlr_table.png)
 
-Much better, am I right? Zettlr's table editor takes all Markdown tables found in your document, converts them into an HTML representation with editable content cells, and displays these instead of the underlying Markdown. This completely hides the Markdown source, but as we said, in this case this approach is much more favourable.
+Much better, am I right? Zettlr's table editor takes all Markdown tables found in your document, converts them into an HTML representation with editable content cells, and displays these instead of the underlying Markdown. This completely hides the Markdown source, but as we said, in this case this approach is much more favourable. The table editor can be opened by selecting the `Insert Table` item in the context menu.
 
```

Comment:
I don't know if it is called "context menu" in macOS

Reply:
I didn't even know I put that item there :D But yes, it's the context menu!

Reply:
Buuuut it's not quite correct: The Table editor can be enabled/disabled in the settings and will then allow you better editing of the Markdown tables, so maybe we should adapt the wording here …

Reply:
Will check the settings and update this (probably sometime tomorrow)

Reply:
Is this any better?

Reply:
Hmm … not sure; I'd opt for making it clear that these are two different things:

Feature one: You can insert Markdown tables using the context menu or (recommended) the formatting menu.

Feature two: With the Table Editor enabled, this allows you to better write Markdown tables. The Table Editor is always active and will pre-render your Markdown tables with some controls.

Something like this, because inserting a table does not "enable" the table editor, rather you insert a table, and then the Table Editor recognizes it and transforms it into a better editable table for you!

Reply:
Oh, I see... I misunderstood the whole thing :) Will fix.

Reply:
Take 3...